vars.tf

```diff
@@ -332,4 +342,14 @@ variable "s3_tags" {
   default     = {}
 }
 
+variable "enable_api_gateway" {
+  type        = bool
+  description = "allow to create api-gateway"
+  default     = false
+}
+
+variable "api_gateway_name" {
```

Comment:
Should be computed base of the name for the application, look at the event bus as an example

Reply:
fixed